agenta-backend/agenta_backend/routers/evaluation_router.py

```diff
@@ -386,6 +456,125 @@ async def fetch_results(
         results = await fetch_results_for_auto_ai_critique(evaluation_id)
         return {"results_data": results}
 
+    elif evaluation.evaluation_type == EvaluationType.custom_code_run:
+        results = await fetch_average_score_for_custom_code_run(evaluation_id)
+        return {"avg_score": results}
+
+
+@router.post("/custom_evaluation/store/")
+async def store_custom_evaluation(
```

Comment:
let's please use create instead of store

Reply:
Sure, sure.